index.html

```diff
@@ -0,0 +1,1230 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter Cloned</title>
+    <link rel="stylesheet" href="main.css">
+</head>
+
+<body>
+    <div id="container">
+
+        <!-- Left pane -->
+        <nav class="nav-pane">
+            <div class="logo-area">
+                <svg viewBox="0 0 24 24" aria-hidden="true">
+                    <g>
+                        <path
```

Comment:
Consider keeping each SVG info in a seperate file in your project to make the HTML more readable